aspnetcore/security/authentication/accconfirm.md

```diff
@@ -145,7 +145,13 @@ Run the web app, and test the account confirmation and password recovery flow.
 * Enter the email you used to register the account.
 * An email with a link to reset your password is sent. Check your email and click the link to reset your password. After your password has been successfully reset, you can sign in with your email and new password.
 
-## Change email and activity timeout
+<a name="resend"></a>
+
+## Resend email confirmation
+
+In ASP.NET Core 5.0 RC1 and later, select the **Resend email confirmation** link on the **Login** page.
```

Comment:
Its been enabled probably since preview 6 if not earlier, so we can maybe drop the RC1?